proposals/fit_curve.json

```diff
@@ -1,38 +1,33 @@
 {
     "id": "fit_curve",
     "summary": "Curve fitting",
-    "description": "Use non-linear least squares to fit a model function `y = f(x, parameters)` to data.\n\nThe process throws an `InvalidValues` exception if invalid values are encountered. Valid values are finite numbers (see also ``is_valid()``).",
+    "description": "Use non-linear least squares to fit a model function `y = f(x, parameters)` to data.",
     "categories": [
-        "cubes",
+        "arrays",
         "math"
     ],
     "experimental": true,
     "parameters": [
         {
             "name": "data",
-            "description": "A data cube.",
+            "description": "An array.",
             "schema": {
-                "type": "object",
-                "subtype": "datacube"
+                "type": "array",
+                "items": {
+                    "type": "number"
+                }
```

Comment:
I think we are still missing a source for the `x` values here. Possible solutions:
- `data` is a labeled array, with x values as labels
- add another array parameter for the `x` values (and rename `data` to `y` for clarity?)

Reply:
A labeled array is what was used already, so I vote for that.

Reply:
Updated.